tests/python_functional/src/common/blocking.py

```diff
@@ -37,7 +37,7 @@ def wait_until_true_custom(func, args=(), timeout=DEFAULT_TIMEOUT):
         if result:
             return result
         time.sleep(POLL_FREQ)
-    return False
+    return result
```

Comment:
Thanks, +1

Reply:
After we discussed this IRL again, we decided that the change is ok. The reason of this change is that it would be good if the result type is consistent.